shared/teams/add-members-wizard/confirm.tsx

```diff
@@ -218,12 +217,11 @@ const RoleSelector = ({disabledRoles, memberCount}: RoleSelectorProps) => {
         open={showingMenu}
         presetRole={storeRole}
         onCancel={storeRole === role ? () => setShowingMenu(false) : undefined}
-        selectedRole={role || 'writer'}
-        onSelectRole={onSelectRole}
         onConfirm={onConfirmRole}
-        confirmLabel="Save"
+        confirmLabel="Save as"
```

Comment:
this is awkward. Maybe take either a string e.g. `'save'` or a function e.g. ```role => `Let in as ${pluralize("role", count)}` ``` and then get rid of count param?